python/tests/test_example.py

```diff
@@ -219,14 +218,12 @@ def test_example(
         code = add_helicone_headers(code)
         filepath.write_text(code, encoding="utf-8")
 
-    cwd = example.get("cwd", None)
     proc = subprocess.Popen(  # pylint: disable=consider-using-with
```

Comment:
While removing the `cwd` parameter simplifies the code, it might be worth adding a comment explaining why this change was made and confirming that all tests still work correctly with absolute paths. This helps future maintainers understand the rationale behind the change.